src/main.js

```diff
@@ -1,5 +1,6 @@
 import Vue from 'vue'
 import App from './App.vue'
+import './styles/typography.scss'
```

Comment:
Include `colors.scss` as well? See PR #11 
```suggestion
import './styles/typography.scss'
import './styles/colors.scss'
```